src/stirling/source_connectors/socket_tracer/socket_trace_connector_test.cc

```diff
@@ -1815,5 +1819,55 @@ TEST_F(SocketTraceConnectorTest, HTTP2OldStream) {
   EXPECT_GT(record_batch[kHTTPLatencyIdx]->Get<types::Int64Value>(3), 0);
 }
 
+// TODO(ddelnano): Add more mux test cases here
+TEST_F(SocketTraceConnectorTest, MuxBasic) {
```

Comment:
For new protocols, we don't need to add a test here. MySQL and CQL have some tests here for historical reasons, but we need to remove those as well (sorry if that led you astray in any way).

Anyways, we want this file to focus on the functionality of the socket tracer, for things that are protocol independent. We do use HTTP in many tests here, but that's just because it's an easy protocol to work with...we really don't care about the specifics of the protocol here.

The upshot is that we don't want to add protocol specific tests here. If we have good coverage in the protocol parsers itself, and through the "bpf_tests" (which provide more end-to-end covrage), we should be okay. Otherwise, if we add protocol specific tests here, this file will get very long.

Reply:
Thanks for the context on these tests. I'll remove this.